java/src/jmri/SpeedStepMode.java

```diff
@@ -21,11 +21,14 @@
 @javax.annotation.concurrent.Immutable
 public enum SpeedStepMode {
     Unknown("unknown"),
+    // NMRA DCC standard speed step modes.
     SpeedStepMode128("128"),
```

Comment:
A suggestion: instead of this step being `SpeedStepMode.SpeedStepMode128` when referenced, changing this to read `SpeedStepMode.NMRA_DCC_128` would make it explicitly clear what this mode is. You might want to do that for every step; i.e. have these steps:
- `UNKNOWN`
- `NMRA_DCC_128`
- `NMRA_DCC_28`
- `NMRA_DCC_27`
- `NMRA_DCC_14`
- `MOTOROLA_28`
- `TMCC_32`

Per https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html the keys in enums should be all upper case  (the keys are the steps in this case). Also per that document, a key can have multiple values, so you could provide both the number of steps and the factor for the increments in this enum (doing that could simplify abstract throttle as well).

Reply:
I was trying to keep the original names, but I think this is better. Done.